tests/contrib/subjects/test_subjects_schema.py

```diff
@@ -39,3 +40,35 @@ def test_invalid_missing_field(subject_full_data):
     del invalid["subject"]
     with pytest.raises(ValidationError):
         SubjectSchema().load(invalid)
+
+
+#
+# SubjectRelationSchema
+#
+def test_valid_id():
+    valid_id = {
+        "id": "test",
+    }
+    assert valid_id == SubjectRelationSchema().load(valid_id)
+
+
+def test_valid_subject():
+    valid_subject = {
+        "subject": "Entity One"
+    }
+    assert valid_subject == SubjectRelationSchema().load(valid_subject)
+
+
+def test_valid_full():
```

Comment:
The subjects vocabulary store multiple subject vocabularies (MeSH, Dewey, OECD Fields of Science) so the scheme is used to distinguish between which one they belong to.

Reply:
@lnielsen Agree, but wouldn't that case be the `id` case? i.e. when it is from a "controlled vocabulary"? when using a *custom* one, allowing scheme means that it can be "added" (it does not add to db but stays on the record), to one of those (i.e. I can add a subject a "fake" it to be a MeSH term).

Reply:
As an example, of a dummy record in inveniordm demo instance:

Payload:

```json
"subjects": [
            {
                "subject": "test-non-exists"
            },
            {
                "id": "http://www.oecd.org/science/inno/38235147.pdf?2.1",
                "scheme": "FOS",
                "subject": "Civil engineering"
            }
        ]
```

Deposit form:

![Screenshot 2022-03-07 at 08 53 55](https://user-images.githubusercontent.com/6756943/156990159-f93dec65-849e-4472-a596-021ea1463d77.png)